crates/sui-types/src/crypto.rs

```diff
@@ -139,6 +139,9 @@ pub struct PublicKeyBytes(
 );
 
 impl PublicKeyBytes {
+    pub const MIN: Self = PublicKeyBytes([u8::MIN; dalek::PUBLIC_KEY_LENGTH]);
+    pub const MAX: Self = PublicKeyBytes([u8::MAX; dalek::PUBLIC_KEY_LENGTH]);
```

Comment:
What's the purpose of these?

Reply:
It doesn't look like they're used?

Reply:
Oh, whoops, I was using a BTreeMap in an earlier revision, will revert.